### PR TITLE
Components

### PR DESCRIPTION
**Pull request by ahzia**
In this project, I continue with the development of the Math Magicians app.
I will develop a React component that will hold the core functionality: a calculator.

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+          <div className="d-flex m-1">
+            <div className="btn-rectingle btn-zero text-first d-flex justify-content-center align-items-center">0</div>
+            <div className="btn-rectingle text-white d-flex justify-content-center align-items-center">.</div>
+            <div className="btn-rectingle text-white d-flex justify-content-center align-items-center">=</div>
+          </div>
+        </div>
+      </div>
+    );
+  }
+}
```
Comment (barackm) [inline src/components/Calculator.js]: - [x] Wow you have done an incredible job in your design, the calculator looks really awesome 👏 , However, it seems that for now, the project requirements state that we should stick to the given design, see screenshot below 👇 
image

- This is how yours looks like for now 👇 
image

As you can see there is a big difference, I would suggest that you start working with the design provided in the project requirements, then at the end of the last project in this series, you can change the design to match what you really like 😉 , but congratulations so far, the design was awesome 👏

**Review by barackm — CHANGES_REQUESTED** [review]
Hi @ahzia, 👋 

You have done an amazing job so far! 👏👏👏
However, there are some issues that you still need to work on to go to the next project. But remember you are almost there 😉  😉 !

# STATUS: Required Changes ♻️

- [ ] [OPTIONAL] Kindly let's fix the demo link in the README file since for now it seems to be broken, I am getting a 404 error (Page not found). 👍 

### _Check other comments under the review._ 👇 


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js (hunk 1 above)
Comment (ahzia) [reply src/components/Calculator.js]: @barackm  Thank you for the Review,
The Problem with the live link is that I have not deployed the project yet, and I will do it next milestones, I deleted the live link part and wrote coming soon instead.
About the Design, My Design has the same structure as the wireframe, Every component is in its own place, the only difference is the color which I don't think will affect the Project.

Comment (barackm) [reply src/components/Calculator.js]: Hi @ahzia, no worries if the project is not yet deployed feel free to do so, for now, I'm changing that requirement to be optional. sorry for the delay 😉

**Comment by Temmarie** [comment]
Hi @ahzia :wave: 

_This is an additional review as a result of the second opinion request._

Thanks for making this request. You complained about the previous code reviewer asking you to add a live link and to change your design to match the wireframe.
Adding the Livelink as you pointed out will be required in the future milestones although it is recommended for you to deploy it now.  However, in the design aspect, you do have to make the change and adjust the project to match the provided design, and as you said, it is just the colours so it shouldn't take you much time. 

image

Kindly make this change and request a new code review request. I hope this helps to resolve the issue.

Happy coding and keep rocking! 🚀

**Comment by ahzia** [comment]
Thank You @Temmarie for Reviewing the Project, although I think My design didn't have any issue with project Requirement, I made changes so it look just like the wireframe.

**Review by wandji20 — APPROVED** [review]
## Hi @ahzia  :raised_hands:

##### You have done exceptionally well, and have met all requirements for this milestone:+1:

# Status: Approved 👍 :partying_face: 
Your project has been approved, however, I have left some comments to help improve your project.

## Congratulations :+1:  :fireworks: 
## Optional
- [ optional ] It will be nice to ignore all files that are not relevant to run this project(eg .lintrc, .stylelintrc.json, etc).

_Go ahead and merge this branch. Also, don't forget to submit the project completion form.
Good luck with your next project :mechanical_arm:  :slightly_smiling_face:_ 

 If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ahzia** [comment]
@wandji20  Thanks for approving the Project I will merge my branch with "develop".
About your optional comment, I cant find these files you mentioned in my project.

**Pull request merged**